README.md

```diff
@@ -2,4 +2,4 @@
 
 ## Przykład wspólnej pracy nad projektem do którego mamy dostęp read only
 
-### Aron tu był
\ No newline at end of file
+### Radek tu był
```

Comment:
Jeśli dobrze rozumiem to usunąłeś mnie a dodałeś siebie, popraw tak żebyśmy obaj zostali na liście